arcgis-ios-sdk-samples/Layers/Query with CQL filters/QueryWithCQLFilters.storyboard

```diff
@@ -0,0 +1,269 @@
+                                            <subviews>
+                                                <datePicker contentMode="scaleToFill" enabled="NO" contentHorizontalAlignment="center" contentVerticalAlignment="center" datePickerMode="date" style="wheels" useCurrentDate="NO" translatesAutoresizingMaskIntoConstraints="NO" id="6LQ-al-yZa">
+                                                    <rect key="frame" x="0.0" y="0.0" width="414" height="44"/>
+                                                    <date key="date" timeIntervalSinceReferenceDate="347691600.57075">
+                                                        <!--2012-01-08 05:00:00 +0000-->
+                                                    </date>
+                                                    <date key="minimumDate" timeIntervalSinceReferenceDate="-978267600">
+                                                        <!--1970-01-01 11:00:00 +0000-->
+                                                    </date>
+                                                    <connections>
+                                                        <action selector="datePickerAction:" destination="GDg-hx-dhZ" eventType="valueChanged" id="YHh-W9-Adr"/>
+                                                    </connections>
+                                                </datePicker>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstItem="6LQ-al-yZa" firstAttribute="top" secondItem="aDh-Nr-jw9" secondAttribute="top" id="0Wj-0p-UrK"/>
+                                                <constraint firstAttribute="bottom" secondItem="6LQ-al-yZa" secondAttribute="bottom" id="74L-lN-s70"/>
+                                                <constraint firstAttribute="trailing" secondItem="6LQ-al-yZa" secondAttribute="trailing" id="EqY-NN-p4D"/>
+                                                <constraint firstItem="6LQ-al-yZa" firstAttribute="leading" secondItem="aDh-Nr-jw9" secondAttribute="leading" id="tmj-qB-MO5"/>
+                                            </constraints>
+                                        </tableViewCellContentView>
+                                    </tableViewCell>
+                                </cells>
+                            </tableViewSection>
+                        </sections>
+                        <connections>
+                            <outlet property="dataSource" destination="GDg-hx-dhZ" id="2n9-IR-0fL"/>
+                            <outlet property="delegate" destination="GDg-hx-dhZ" id="d6V-74-n5T"/>
+                        </connections>
+                    </tableView>
+                    <navigationItem key="navigationItem" title="CQL Filters" id="iQj-FT-f4r">
+                        <barButtonItem key="leftBarButtonItem" title="Cancel" id="AK8-Ni-KMZ">
```

Comment:
This should be a system Cancel button.

![image](https://user-images.githubusercontent.com/2257493/122112030-3ff9dd80-cdd5-11eb-8b8e-a26877c1f770.png)